subprojects/gradle/build-checks/src/main/kotlin/com/avito/android/build_checks/RootProjectChecksExtension.kt

```diff
@@ -38,11 +39,11 @@ public open class RootProjectChecksExtension : BuildChecksExtension() {
     public fun gradleProperties(action: Action<GradleProperties>): Unit =
         register(GradleProperties(), action)
 
-    @Deprecated("remove it, has no effect, use versions lock")
+    @Deprecated("remove it, has no effect")
     public fun moduleTypes(action: Action<ModuleTypes>): Unit =
         register(ModuleTypes(), action)
 
-    @Deprecated("remove it, has no effect, use versions lock")
+    @Deprecated("remove it, has no effect")
     public fun incrementalKapt(action: Action<IncrementalKapt>): Unit =
```

Comment:
We keep extensions to preserve backward compatibility for clients.
Deprecated methods are easy to miss, especially in groovy files.
It'd be good to log a warning about usage of deprecated API like AGP and other plugins do.

Reply:
done